lib/calendar.dart

```diff
@@ -61,7 +67,7 @@ class _MyHomePageState extends State<MyHomePage> {
       appBar: AppBar(
         centerTitle: true,
         title: Text(
-          widget.title,
```

Comment:
This is a temporary change. We can revert this to widget.title in the final version. the effect of this change just makes the appbar at the top read "Inkling" instead of what it previously said.